pkg/gofr/cron_test.go

```diff
@@ -606,3 +606,25 @@ func TestCron_parsePart(t *testing.T) {
 		})
 	}
 }
+
+func TestCron_WithTimezone(t *testing.T) {
+	tests := []struct {
+		name     string
+		option   func(*Crontab)
+		expected *time.Location
+	}{
+		{
+			name:     "Factory with timezone option",
+			option:   WithTimezone(time.UTC),
+			expected: time.UTC,
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			got := NewCron(nil, test.option)
+			got.ticker.Stop()
+			assert.Equal(t, test.expected, got.location)
+		})
+	}
+}
```

Comment:
Please also add a test without an option, something like this

```suggestion
func TestCron_NewCron(t *testing.T) {
	tests := []struct {
		name     string
		options   []func(*Crontab)
		expected *time.Location
	}{
		{
			name:     "Factory without options",
			expected: time.Local,
		},
		{
			name:     "Factory with timezone option",
			options:   []func(*Cromtab){WithTimezone(time.UTC)},
			expected: time.UTC,
		},
	}

	for _, test := range tests {
		t.Run(test.name, func(t *testing.T) {
			got := NewCron(nil, test.options...)
			got.ticker.Stop()
			assert.Equal(t, test.expected, got.location)
		})
	}
}
```

Reply:
Please also add a test with time.Local and America/newyork as you did in the documentation